src/flb_engine.c

```diff
@@ -522,6 +524,9 @@ int flb_engine_start(struct flb_config *config)
     }
     config->evl = evl;
 
+    /* Create the bucket queue (FLB_ENGINE_PRIORITY_COUNT priorities) */
+    evl_bktq = mk_bucket_queue_create(FLB_ENGINE_PRIORITY_COUNT);
```

Comment:
Not deleting this memory for now. Move pqueue to config potentially handle delete in config clean up function potentially